lib/sendWelcomeEvent.ts

```diff
@@ -29,7 +30,7 @@ export const sendWelcomeEventToDialogFlow = async (app: IApp, read: IRead,  modi
         const response: IDialogflowMessage = await Dialogflow.sendRequest(http, read, modify, rid, event, DialogflowRequestType.EVENT);
         await createDialogflowMessage(rid, read, modify, response, app);
     } catch (error) {
-        console.error(`${Logs.DIALOGFLOW_REST_API_ERROR} ${error.message}`);
+        console.error(`${Logs.DIALOGFLOW_REST_API_ERROR} ${getError(error)}`);
```

Comment:
@Shailesh351 @ItzzRitik the error message is still sending an empty dict for ${getError(error)}

thoughts?

Reply:
@Shailesh351 found this:  https://stackoverflow.com/questions/18391212/is-it-not-possible-to-stringify-an-error-using-json-stringify

@ItzzRitik will this help you fix?